### PR TITLE
Updated basic_bot.py

### PR DESCRIPTION
**Pull request by itzsimpleboii8282**
Added the ping command and a simple error handler(Missing Argument).

## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Comment by itzsimpleboii8282** [comment]
Just added a ping command in basic bot, with a error handler

File: examples/basic_bot.py [hunk 1]
```diff
@@ -48,7 +48,16 @@ async def repeat(ctx, times: int, content='repeating...'):
     """Repeats a message multiple times."""
     for i in range(times):
         await ctx.send(content)
-
+@bot.command()
+async def ping(ctx):
+    """A simple ping command!""" 
+    embed = discord.Embed(title="Pong!", description=f"```{round(bot.latency * 1000)} ms```", color = discord.Color.blue())
```
Comment (Vexs) [inline examples/basic_bot.py:54]: Lib minimum version does not support f-strings

File: examples/basic_bot.py [hunk 2]
```diff
@@ -48,7 +48,16 @@ async def repeat(ctx, times: int, content='repeating...'):
     """Repeats a message multiple times."""
     for i in range(times):
         await ctx.send(content)
-
+@bot.command()
+async def ping(ctx):
+    """A simple ping command!""" 
+    embed = discord.Embed(title="Pong!", description=f"```{round(bot.latency * 1000)} ms```", color = discord.Color.blue()) 
+    await ctx.send(embed=embed)
+@bot.event
```
Comment (Vexs) [inline examples/basic_bot.py:56]: Missing newlines.

Comment (Vexs) [reply examples/basic_bot.py:56]: Event also shouldn't be plopped in the middle of commands.

File: examples/basic_bot.py [hunk 3]
```diff
@@ -48,7 +48,16 @@ async def repeat(ctx, times: int, content='repeating...'):
...
-
+@bot.command()
+async def ping(ctx):
+    """A simple ping command!""" 
+    embed = discord.Embed(title="Pong!", description=f"```{round(bot.latency * 1000)} ms```", color = discord.Color.blue()) 
+    await ctx.send(embed=embed)
+@bot.event 
+async def on_command_error(ctx, error):
+	error = getattr(error, 'original', error)
+	if isinstance(error, commands.MissingRequiredArgument):
```
Comment (Vexs) [inline examples/basic_bot.py:59]: This eats every other error. No-go.

File: examples/basic_bot.py [hunk 4]
```diff
@@ -48,7 +48,16 @@ async def repeat(ctx, times: int, content='repeating...'):
...
         await ctx.send(content)
-
+@bot.command()
+async def ping(ctx):
+    """A simple ping command!""" 
+    embed = discord.Embed(title="Pong!", description=f"```{round(bot.latency * 1000)} ms```", color = discord.Color.blue()) 
+    await ctx.send(embed=embed)
+@bot.event 
+async def on_command_error(ctx, error):
+	error = getattr(error, 'original', error)
```
Comment (Vexs) [inline examples/basic_bot.py:58]: Needs comments explaining the purpose of this.

File: examples/basic_bot.py [hunk 5]
```diff
@@ -48,7 +48,16 @@ async def repeat(ctx, times: int, content='repeating...'):
     """Repeats a message multiple times."""
     for i in range(times):
         await ctx.send(content)
-
+@bot.command()
+async def ping(ctx):
```
Comment (Vexs) [inline examples/basic_bot.py:52]: This command seems slightly redundant given what else is in this example.

**Review by Vexs — CHANGES_REQUESTED** [review]

File: examples/basic_bot.py (hunk 4 above)
Comment (itzsimpleboii8282) [reply examples/basic_bot.py:58]: This was just to add a error handler.

File: examples/basic_bot.py (hunk 3 above)
Comment (itzsimpleboii8282) [reply examples/basic_bot.py:59]: Then, other handlers could be added too, in this way it'll not eat other errors.

Comment (numbermaniac) [reply examples/basic_bot.py:59]: Consider adding an `else:` block to raise the error if it's of another type.

File: examples/basic_bot.py (hunk 4 above)
Comment (kaylynn234) [inline examples/basic_bot.py:58]: The quotes used here are inconsistent with the rest of this example.

Comment (numbermaniac) [reply examples/basic_bot.py:58]:
```suggestion
	error = getattr(error, "original", error)
```

File: examples/basic_bot.py (hunk 3 above)
Comment (itzsimpleboii8282) [reply examples/basic_bot.py:59]: okai, thx, will add...

**Comment by itzsimpleboii8282** [comment]
Gotta, close this pull request, and then open a new neat one.

**Pull request closed without merging**